airlock/enums.py

```diff
@@ -74,9 +74,22 @@ class RequestFileDecision(Enum):
     CONFLICTED = "CONFLICTED"
     INCOMPLETE = "INCOMPLETE"
 
-    def description(self):  # pragma: nocover
+    def description(self):
         return self.name.replace("_", " ").title()
 
+    def reason(self):
+        match self:
+            case RequestFileDecision.INCOMPLETE:
+                return "Overall decision will be displayed after two independent output checkers have submitted their review of all output files"
+            case RequestFileDecision.APPROVED:
+                return "Two independent output checkers have approved this file"
+            case RequestFileDecision.CHANGES_REQUESTED:
+                return "Two independent output checkers have requested changes to this file"
+            case RequestFileDecision.CONFLICTED:
+                return "Output checkers are conflicted in their review of this file"
```

Comment:
Small suggested tweak: "Output checkers have reviewed this file and disagree about whether it should be released".